src/components/CreateButton.tsx

```diff
@@ -164,6 +162,28 @@ export const CreateButton = () => {
 
         if (!valid()) return;
 
+        let claimAddress = onchainAddress();
+
+        if (assetReceive() === RBTC) {
+            const [balance, gasPrice] = await Promise.all([
+                signer().provider.getBalance(await signer().getAddress()),
+                signer()
+                    .provider.getFeeData()
+                    .then((data) => data.gasPrice),
+            ]);
+            log.debug("RSK balance", balance);
+
+            const balanceNeeded = gasPrice * GasNeededToClaim;
```

Comment:
when deploying for the first time should we also account for `GasNeededToDeploy` here?